src/components/Message/MessageSimple/MessageActionSheet.js

```diff
@@ -0,0 +1,108 @@
+import React, { useContext } from 'react';
+import styled from '@stream-io/styled-components';
+import PropTypes from 'prop-types';
+import { ActionSheetCustom as ActionSheet } from 'react-native-actionsheet';
```

Comment:
this rename just for consistency with the other ActionSheet?

Reply:
yeah, and also we're calling the new custom prop ActionSheet